types/google-cloud__storage/index.d.ts

```diff
@@ -5,7 +5,340 @@
+        upload(localPath: string, options?: Storage.UploadOptions): Promise<[File]>;
+    }
+
+    /**
+     * Bucket configuration.
+     */
+    interface BucketConfig {
+        coldline?: boolean;
+        dra?: boolean;
+        location?: string;
+        multiRegional?: boolean;
+        nearline?: boolean;
+        regional?: boolean;
+        versioning?: {
+            enabled?: boolean
+        };
+    }
+
+    /**
+     * Options for getting a file.
+     */
+    interface BucketFileOptions {
+        generation?: string | number;
+    }
+
+    /**
+     * Options for getting a bucket.
+     */
+    interface BucketGetOptions {
+        autoCreate?: boolean;
+    }
+
+    /**
+     * Bucket metadata.
+     */
+    interface BucketMetadata {
+        etag?: string;
+        id?: string;
+        kind?: string;
+        location?: string;
+        metageneration?: string;
+        name?: string;
+        projectNumber?: string;
+        selfLink?: string;
+        storageClass?: string;
+        timeCreated?: string;
+        updated?: string;
+    }
+
+    /**
+     * The options for making the bucket private.
+     */
+    interface BucketPrivacyOptions {
+        includeFiles?: boolean;
+        force?: boolean;
+    }
+
+    /**
+     * Query a bucket.
+     */
+    interface BucketQuery {
+        autoPaginate?: boolean;
+        delimiter?: string;
+        prefix?: string;
+        maxApiCalls?: number;
+        maxResults?: number;
+        pageToken?: string;
+        versions?: boolean;
+    }
+
+    /**
+     * Options for specifying the content length range.
+     */
+    interface ContentLengthRange {
+        max?: number;
+        min?: number;
+    }
+
+    /**
+     * A file in the cloud.
+     */
+    class File {
+        constructor(bucket: Bucket, name: string, options?: BucketFileOptions);
+        acl: Acl;
+        copy(destination: string | Bucket | File): Promise<[File, ApiResponse]>;
+        createReadStream(options?: ReadStreamOptions): ReadStream;
+        createWriteStream(options?: WriteStreamOptions): WriteStream;
+        delete(): Promise<[ApiResponse]>;
+        download(options?: DownloadOptions): Promise<[Buffer]>;
+        exists(): Promise<[boolean]>;
+        get(): Promise<[File, ApiResponse]>;
+        getMetadata(): Promise<[FileMetadata, ApiResponse]>;
+        getSignedPolicy(options?: SignedPolicyOptions): Promise<[SignedPolicy]>;
+        getSignedUrl(config?: SignedUrlConfig): Promise<[string]>;
+        makePrivate(options?: FilePrivateOptions): Promise<[ApiResponse]>;
+        makePublic(): Promise<[ApiResponse]>;
+        move(destination: string | Bucket | File): Promise<[File, ApiResponse]>;
+        name: string;
+        save(data: string, options?: WriteStreamOptions): Promise<void>;
+        setEncryptionKey(encryptionKey: string | Buffer): File;
+        setMetadata(metadata: FileMetadata): Promise<[ApiResponse]>;
+        metadata?: FileMetadata;
```

Comment:
[`setStorageClass`](https://googlecloudplatform.github.io/google-cloud-node/#/docs/storage/1.2.0/storage/file?method=setStorageClass) seems to be missing.